### PR TITLE
Render default component if not enrolled in ABTest

### PR DESCRIPTION
**Pull request by mattrheault**
If I have an experiment that I want to show conditionally on some value, then I would assume that if a user is not enrolled, then they should see the children of the ```Default``` component.

**EX:** Here we should see the Default component render if ```shouldEnroll``` is false.
```javascript
render() {
  const shouldEnroll = true || false;

  return(
    <ABTest experiment={experiment} on='foo' shouldEnroll={shouldEnroll}>
      <When value='bar'>
        {...}
      </When>
      <Default>
        {...}
      </Default>
    </ABTest>
  );
}
```

---
@rawls238 @gusvargas

### CODE REVIEW

File: __tests__/testAbTest.js [hunk 1]
```diff
@@ -110,4 +110,25 @@ describe('Test experiment component', () => {
...
+          <div className='default-div'>
+            Test
+          </div>
+        </ReactExperiments.Default>
+      </ReactExperiments.ABTest>
+    );
+    expect(TestUtils.scryRenderedDOMComponentsWithClass(
+      experimentComponent,
+      'default-div'
+    ).length).toBe(1);
```
Comment (rawls238) [inline __tests__/testAbTest.js]: can we also verify that the log length is 0?

**Comment by rawls238** [comment]
apart from adding that test, this looks good to me.

**Comment by rawls238** [comment]
Also I should really finish setting up travis-ci for this...

**Comment by mattrheault** [comment]
@rawls238 @gusvargas 

Closing this. We talked it out and decided that ```shouldEnroll``` should be deprecated and abstracted away to the planout experiment/namespace class definition.

**Pull request closed without merging**